.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Please fill out this information before submitting a bug:
+
+  - `ruby --version`: ...
+  - `gem --version`: ...
```

Comment:
maybe instead of `ruby` and `gem` versions `gem env` would provide more information? it includes such important information as configuration, home, path, shell path - I could read a lot from it.